plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/application/TemplateListPage.java

```diff
@@ -151,9 +154,15 @@ public void widgetSelected(SelectionEvent e) {
 						try {
 							model.setTemplateFileName(file);
 							return;
-						}catch(ResourceFactoryException | ClassCastException err) {
+						}catch(ClassCastException err) {
```

Comment:
@maxandersen only thing I can think of is to shield the user from a long trace they don't understand which is the reason it gets logged.  In hind site, I guess that's the reason for the details button?  To present the user with a simple message and an opportunity to see more extensive information?